packages/core/src/api.ts

```diff
@@ -47,9 +47,7 @@ export function getEnvelopeEndpointWithUrlEncodedAuth(dsn: DsnComponents, tunnel
 export function getReportDialogEndpoint(
   dsnLike: DsnLike,
   dialogOptions: {
```

Comment:
I wonder if this should be typed with `ReportDialogOptions`

Reply:
I moved the type to core so we can reuse this here!